cmds/dhclient/dhclient.go

```diff
@@ -158,15 +160,25 @@ func dhclient4(iface netlink.Link, numRenewals int, timeout time.Duration, retry
 				routerName := net.IP(gwData).String()
 				debug("routerName %v", routerName)
 				r := &netlink.Route{
-					Dst:       &net.IPNet{IP: packet.GIAddr()},
 					LinkIndex: iface.Attrs().Index,
 					Gw:        net.IP(gwData),
 				}
 
-				if err := netlink.RouteReplace(r); err != nil {
+				if err := netlink.RouteAdd(r); err != nil {
+					debug("RouteAdd %v returned %v", r, err)
 					return fmt.Errorf("%s: add %s: %v", iface.Attrs().Name, r.String(), routerName)
 				}
 			}
+			if ip, ok := o[dhcp4.OptionDomainNameServer]; ok {
+				rc := ""
+				// multiples of 4 octets.
+				for i := 0; i < len(ip); i += 4 {
```

Comment:
Aaaactually... I'm gonna be a pain in the ass for a second.

```go
for i := 0; i < len(ip); i += 4 {
  rc = fmt.Sprintf("%s nameserver %s\n", net.IP(ip[i:i+4]))
}
```

Reply:
Thanks, I could not for the life of me recall how to do that. Change coming again.